src/recordmerge.jl

```diff
@@ -0,0 +1,13 @@
+"""
+    recordmerge(record; fields_to_merge...)
+
+Return an `AbstractRecord` by merging the fields of `record` and `fields_to_merge` together.
+
+See also: [`Tables.rowmerge`](@ref).
```

Comment:
```suggestion
```
The Documenter `@ref` doesn't work for things that aren't included in the same output document.

Reply:
Interesting. Surprised I haven't run into that before. I'd prefer to still have a mention here so maybe just without the link?

```suggestion

See also: `Tables.rowmerge`.
```